distribution/test/itests/test-itests-dependencies-app/src/main/filtered-resources/features.xml

```diff
@@ -36,7 +36,7 @@
         <bundle>mvn:ddf.test.itests/test-itests-common/${project.version}</bundle>
 
         <bundle>mvn:ddf.test.thirdparty/restito/${project.version}</bundle>
-        <feature>broker-app</feature>
+        <feature>camel-sjms</feature>
```

Comment:
Is this feature only needed for `TestMessageBroker` and not all of the itests? Can it be started in that test class instead?